upgrades.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Upgrades
+description: This document describes how to upgrade Flipt.
+---
+
+Flipt aims to provide an upgrade process which requires zero downtime with minimal steps.
+Currently, Flipt supports four relational database backends and uses a migration process to manage their schema.
+
+Each new release of Flipt, may or may not include updates to database migrations.
+The release notes on GitHub for each Flipt release should signify whether it does or not.
+
+You can find the releases and release notes on [GitHub](https://github.com/flipt-io/flipt/releases).
+
+Before upgrading your Flipt instances to a new version, you need to run the `flipt migrate` command against your database.
+The Flipt binary is self-contained and has all necessary migrations baked in.
+You should use the version of Flipt you're attempting to upgrade to. 
+
+<Note>
+If you attempt to upgrade to a new version of Flipt before running migrations, then Flipt will fail fast on startup with a message like the following.
+
+```
+migrations pending, please backup your database and run `flipt migrate`
+```
+</Note>
+
+## Upgrade Process
+
+<Warning>
+We **strongly recommend** that you perform a backup of your Flipt database before doing any migrations.
+While we strive to ensure that Flipt is error free and production ready, we cannot guarantee perfection.
+</Warning>
+
+First, run `flipt -v` to see the version you have installed locally.
```

Comment:
```suggestion
First, run `flipt -v` to see the version you have installed.
```

Maybe leave out the locally part, so it is more general to wherever they are running flipt.